libiocage/lib/helpers.py

```diff
@@ -103,6 +103,79 @@ def validate_name(name):
     return bool(validate.fullmatch(name))
 
 
+# data -> bool | default
+def parse_bool(data, default=False):
+    """
+    try to parse booleans from strings
+
+    On success, it returns the parsed boolean
+    on failure it returns the `default`.
+    By default, `default` is `False`.
+
+    >>> parse_bool("YES")
+    True
+    >>> parse_bool("false")
+    False
+    >>> parse_bool("/opt/android-studio/build.txt")
+    False
+
+    Note that "-" gets a special treatment:
+
+    >>> parse_bool("-")
+    None
+
+    The behavior of the default parameter can be used to create a
+    pass-thru function:
+
+    >>> data = "/opt/android-studio/build.txt"
+    >>> parse_bool(data, data)
+    "/opt/android-studio/build.txt"
```

Comment:
We should choose some more generic sample string, because otherwise we could accidentally show up in search results for android-studio and confuse people. What about `/etc/passwd`?

Can you rename the variable `data` to `default`. That's a bit more intuitive to read.

Reply:
done